sup3r/pipeline/strategy.py

```diff
@@ -253,6 +253,29 @@ def __post_init__(self):
 
         self.preflight()
 
+    def get_min_pad_width(self, model):
+        """Get the padding values applied in the first padding layer of the
+        model. This is used to determine the minimum width of padded slices
+        used to chunk the generator input."""
+        pad_width = (1, 1, 1)
+        if hasattr(model, '_gen'):
+            layers = model._gen.layers
+        elif hasattr(model, 'models'):
+            # multi-step model
+            layers = model.models[0]._gen.layers
+        else:
+            # topography interpolation models don't have a generator network
+            return pad_width
+        for layer in layers:
+            if hasattr(layer, 'paddings'):
+                new_pw = np.max(layer.paddings, axis=1)[1:-1]
+                if len(new_pw) < 3:
+                    new_pw = (*new_pw, 1)
+                pad_width = [
+                    np.max((new_pw[i], pad_width[i])) for i in range(3)
+                ]
+        return pad_width
```

Comment:
This feels like it could be a bit specific to our current model design. What if we migrate to a different padding layer or change the attr in FlexiblePadding? Seems like a pretty rigid tight coupling. Could we just have min_width be an optional user arg that defaults to (1,1,1)?